### PR TITLE
Bump resource validator version

### PR DESCRIPTION
**Pull request by broadbot**
Update resource validator for https://github.com/broadinstitute/leonardo-cron-jobs/pull/4

This is updated manually since automation is broken after I split the project into sub projects

### CODE REVIEW

File: charts/leonardo/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: leonardo
-version: 0.0.3-RC1
+version: 09-15-20
```
Comment (Qi77Qi) [inline charts/leonardo/Chart.yaml:3]: @choover-broad is this version format okay?

Comment (choover-broad) [reply charts/leonardo/Chart.yaml:3]: nope, helm charts need to be semver!

File: charts/leonardo/values.yaml [hunk 2]
```diff
@@ -44,7 +44,7 @@ cronjob:
   # deployments.standalone.name -- Name to use for the default standalone Leonardo deployment
   name: leonardo-resource-validator-cronjob
   imageRepository: us.gcr.io/broad-dsp-gcr-public/resource-validator
-  resourceValidatorImageTag: 0.0.1-rc3
+  resourceValidatorImageTag: fdba243
```
Comment (choover-broad) [inline charts/leonardo/values.yaml:47]: this is fine 👍

File: charts/leonardo/Chart.yaml (hunk 1 above)
Comment (choover-broad) [reply charts/leonardo/Chart.yaml:3]: can split the existing version x.y.z and set it to x.(y+1).0

**Review by choover-broad — CHANGES_REQUESTED** [review]

**Review by choover-broad — APPROVED** [review]
LGTM! 👍

**Comment by Qi77Qi** [comment]
closing in favor of https://github.com/broadinstitute/terra-helm/pull/121

**Pull request closed without merging**